tests/components/androidtv/test_media_player.py

```diff
@@ -285,6 +287,115 @@ def _setup(hass, config):
         assert state.state == STATE_OFF
 
 
+async def test_androidtv_sources(hass):
```

Comment:
A lot of this test code can be consolidated between the Android TV and Fire TV classes.  I submitted a pull request to your repo that accomplishes this.